docs/source/scale/distributed.rst

```diff
@@ -66,15 +66,16 @@ Pick one instance and run the following command to connect to the GraphStorm Doc
 
     docker container exec -it test /bin/bash
 
-In the container environment, users can check the connectivity with the command ``ssh <ip-in-the-cluster> -p 2222``. Please replacing the ``<ip-in-the-cluster>`` with the real IP address in the ``ip_list.txt`` file above, e.g., 
+In the container environment, users can check the connectivity with the command ``ssh <ip-in-the-cluster> -o StrictHostKeyChecking=no -p 2222``. Please replace the ``<ip-in-the-cluster>`` with the real IP address from the ``ip_list.txt`` file above, e.g.,
 
 .. code-block:: bash
 
-    ssh 172.38.12.143 -p 2222
+    ssh 172.38.12.143 -o StrictHostKeyChecking=no -p 2222
 
 If succeeds, you should login to the container in the ``<ip-in-the-cluster>`` instance. 
 
 If not, please make sure there is no limitation of port 2222. 
+You may also have to exchange the keys from inside the GraphStorm Docker containers to allow their communication. For that, copy the keys from the ``/root/.ssh/id_rsa.pub`` from this container to ``/root/.ssh/authorized_keys`` in containers on every machine in your cluster.
```

Comment:
Maybe key exchanging should be a separate step of the instruction? Is there a plan of GraphStorm Docker images to be available for download instead of building them manually? With the current instructions, it seems that every container has different keys as keys are created inside Dockerfile for every machine separately.

Reply:
So far we have our own internal Docker images, but not publicly available.

Thanks for the new explanation added.